ignite/internal/tools/gen-mig-diffs/go.mod

```diff
@@ -0,0 +1,93 @@
+module github.com/ignite/mig-diffs
```

Comment:
Module name should be the path of where go.mod is:
github.com/ignite/cli/(...)/gen-mig-diffs

Go get will not find it otherwise I think.

Reply:
Do you mean to reuse the code? The `go get` will not be found anyway because it is in the internal folder

Reply:
I meant for installing the tool via `go install`. Never tried to install a tool within `internal/` actually, maybe it will indeed won't work 🤔